src/ansiCommandStream.js

```diff
@@ -38,19 +38,22 @@ const theme = {
 };
 
 // use this function to adjust arguments to yarn or npm
-const adjustArguments = arg => {
+const adjustArgument = arg => {
   switch (arg) {
     case 'remove':
       return 'uninstall';
     case 'add':
-      return 'i';
+      return ['install', '--save'];
   }
 };
 
 module.exports = function ansiCommandStream({ args }) {
   const command = hasYarn() ? 'yarn' : 'npm';
   // adjust arguments only if needed (only if it hasn't yarn)
-  const adjustedArguments = hasYarn() ? args : args.map(adjustArguments);
+  const adjustedArguments = hasYarn()
+    ? args
+    : _.flatten(args.map(adjustArgument));
```

Comment:
Could we switch this to import from `"core-js/modules/es7.array.flatten"`?

https://github.com/zloirock/core-js/blob/master/modules/es7.array.flatten.js

And rewrite to `args.map(adjustArgument).flatten()`?

Since this is probably going to end up in the language it might end up being dropped from lodash.

Reply:
I was going to do something like that in a subsequent PR :), so go ahead